src/screens/Assets/WalletView.js

```diff
@@ -48,14 +48,15 @@ import {
   ETH,
 } from 'constants/assetsConstants';
 import { EXCHANGE, SMART_WALLET_INTRO } from 'constants/navigationConstants';
+import { SMART_WALLET_UPGRADE_STATUSES } from 'constants/smartWalletConstants';
 
 import { activeAccountSelector } from 'selectors';
 import { accountBalancesSelector } from 'selectors/balances';
 import { accountCollectiblesSelector } from 'selectors/collectibles';
 import Spinner from 'components/Spinner';
 import Separator from 'components/Separator';
 import EmptyStateParagraph from 'components/EmptyState/EmptyStateParagraph';
-import { DeploymentView, getDeployErrorMessage } from 'components/DeploymentView';
+import DeploymentView, { getDeployErrorMessage } from 'components/DeploymentView';
```

Comment:
not sure if utils would be a better place to store the functions

Reply:
moved